### PR TITLE
Pan the table to move the map split up and down

### LINKED ISSUE #976: Arrival and Departure: Pan the table to move the map split up and down
### Expected behavior

I should be able to hide or show more of the arrival and departure map by panning on the top part of th table.

### Actual behavior

This doesn't exist. Instead, there's only the expand/contract button.

### Steps to reproduce the behavior

Open Arrival and Departure controller, try panning.

### Screenshot of the issue (if applicable)

mock

### PR DESCRIPTION
**Pull request by ualch9**
# **Not ready for merge**
Fixes #976

### CODE REVIEW

**Comment by ualch9** [comment]
sorry this is taking a while, I have been very busy over the past week but I hope to finish by the end of the weekend.

**Comment by aaronbrethorst** [comment]
no prob, know the feeling :)

File: OneBusAway/ui/trip_details/OBAArrivalAndDepartureViewController.m [hunk 1]
```diff
@@ -113,38 +115,33 @@ - (instancetype)initWithArrivalAndDepartureConvertible:(NSObject<OBAArrivalAndDe
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-
+
```
Comment (aaronbrethorst) [inline OneBusAway/ui/trip_details/OBAArrivalAndDepartureViewController.m]: watch these whitespace-only lines please. Xcode has an option to automatically truncate them that I highly recommend enabling.
screen-shot-2017-02-15-at-10 47 37-pm

File: OneBusAway/ui/trip_details/OBAArrivalAndDepartureViewController.m [hunk 2]
```diff
@@ -175,12 +172,54 @@ - (void)viewWillDisappear:(BOOL)animated {
...
 }
 
+- (void)viewWillLayoutSubviews {
+    [super viewWillLayoutSubviews];
+    [self.mapController.view mas_makeConstraints:^(MASConstraintMaker* make) {
+        make.edges.equalTo(self.collapsingHeaderView);
+    }];
+    
+    [self.collapsingHeaderView mas_makeConstraints:^(MASConstraintMaker* mas) {
+        mas.left.equalTo(self.view.mas_left);
```
Comment (aaronbrethorst) [inline OneBusAway/ui/trip_details/OBAArrivalAndDepartureViewController.m]: Masonry has a neat chaining syntax that'll let you collapse the first part down into:

`mas.left.right.and.top.equalTo(self.view)`

**Comment by ualch9** [comment]
@aaronbrethorst thanks for the feedback, I will look into it

**Pull request closed without merging**